From d3bfcc889ca1f59798f25db94e4d56b4105ce268 Mon Sep 17 00:00:00 2001
From: Aswath Govindraju <a-govindraju@ti.com>
Date: Tue, 24 May 2022 10:58:06 +0530
Subject: [tiU21.01 PATCH v2 1/2] Revert "arm: dts: k3-am625-sk-u-boot: Add
 u-boot tag to dt nodes for enabling UHS speed modes in SPL"

This reverts commit 22247f725fd7b8ebc49738db64feefa4edb0ae99.

Instabilities were observed on few SD cards when UHS speed modes were
enabled. Therefore, disable UHS modes and voltage switching by removing the
repective nodes in the device tree.

Signed-off-by: Aswath Govindraju <a-govindraju@ti.com>
---
 arch/arm/dts/k3-am625-sk-u-boot.dtsi | 40 ----------------------------
 1 file changed, 40 deletions(-)

diff --git a/arch/arm/dts/k3-am625-sk-u-boot.dtsi b/arch/arm/dts/k3-am625-sk-u-boot.dtsi
index 16562cf4942b..f2ad332f9f5d 100644
--- a/arch/arm/dts/k3-am625-sk-u-boot.dtsi
+++ b/arch/arm/dts/k3-am625-sk-u-boot.dtsi
@@ -92,10 +92,6 @@
 	u-boot,dm-spl;
 };
 
-&main_gpio0 {
-	u-boot,dm-spl;
-};
-
 &main_i2c0 {
 	u-boot,dm-spl;
 };
@@ -104,18 +100,6 @@
 	u-boot,dm-spl;
 };
 
-&main_i2c1 {
-	u-boot,dm-spl;
-};
-
-&main_i2c1_pins_default {
-	u-boot,dm-spl;
-};
-
-&exp1 {
-	u-boot,dm-spl;
-};
-
 &sdhci0 {
 	u-boot,dm-spl;
 };
@@ -225,30 +209,6 @@
 	u-boot,dm-spl;
 };
 
-&vmain_pd {
-	u-boot,dm-spl;
-};
-
-&vcc_5v0 {
-	u-boot,dm-spl;
-};
-
-&vcc_3v3_sys {
-	u-boot,dm-spl;
-};
-
-&vdd_mmc1 {
-	u-boot,dm-spl;
-};
-
-&vdd_sd_dv {
-	u-boot,dm-spl;
-};
-
-&vdd_sd_dv_pins_default {
-	u-boot,dm-spl;
-};
-
 &dmsc {
 	u-boot,dm-spl;
 	k3_sysreset: sysreset-controller {
-- 
2.17.1

